packages/mobile/src/utils/supportedRegions.ts

```diff
@@ -0,0 +1,19 @@
+import { InquiryAttributes } from 'react-native-persona'
+
+export function isUserRegionSupportedByFinclusive(
```

Comment:
I prefer the way this was implemented before, when it just returned false if the region wasn't supported.

Three reasons not to throw an error for unsupported regions:
- Runtime errors should be reserved for unexpected behavior. A person being from New York is expected, not a reason to sound the alarm!
- helper functions should be self-documenting and have predictable outputs. A function called `isUserRegionSupportedByFinclusive` that returns a boolean should return false when the user is not in a region supported, because that's what the name strongly suggests it would do. 
- in general, exceptions should not be used for control flow

Reply:
still don't think we should throw errors for non-error cases here